### PR TITLE
Create PTLP_RVP_SDW.sh

### PR DESCRIPTION
**Pull request by harajend**
Added Amixer setting for PTL SDW config

### CODE REVIEW

**Comment by sofci** [comment]
Can one of the admins verify this patch?
> reply **test&nbsp;this&nbsp;please** to run this test once

**Comment by marc-hb** [comment]
test this please

**Review by mohsinkh30 — COMMENTED** [review]
verified

File: alsa_settings/PTLP_RVP_SDW.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+set -e
+
+# enable headset playback and capture
+amixer -c sofsoundwire cset name='Headset Mic Switch' on
+amixer -c sofsoundwire cset name='rt722 FU0F Capture Switch' on
+amixer -c sofsoundwire cset name='rt722 FU0F Capture Volume' 50
```
Comment (bardliao) [inline alsa_settings/PTLP_RVP_SDW.sh:6]: Not sure if we need to set 'rt722 FU0F Capture Volume'. Maybe it is better to keep it as the default value?

**Review by bardliao — APPROVED** [review]

File: alsa_settings/PTLP_RVP_SDW.sh (hunk 1 above)
Comment (harajend) [reply alsa_settings/PTLP_RVP_SDW.sh:6]: For the default value of 75, the test fails with the error message 'over amplitude detected.' We reduced the capture volume and tested values between 30 and 50, which worked fine. Hence, we kept the same value here

**Review by kv2019i — APPROVED** [review]

**Pull request merged**